lib/containers/common.pm

```diff
@@ -254,7 +254,7 @@ sub test_container_image {
     # Images from custom registry are listed with the '$registry/library/'
     $image =~ s/^docker\.io\/library\///;
 
-    my $smoketest = qq[/bin/sh -c '/bin/uname -r; /bin/echo "Heartbeat from $image";ps'];
+    my $smoketest = qq[/bin/sh -c '/bin/uname -r; /bin/echo "Heartbeat from $image"'];
```

Comment:
Why remove the `ps` command?

Reply:
IIRC, the `ps` call was added there to double-check that the whole chain of commands is passed correctly into the container. Not really necessary, but nice to have from my point of view and mainly the call was buggy before so it was an easy review check.

Reply:
Because of an error in `journal_check.pm`:
```
Aug 08 15:59:40.737886 localhost.localdomain testing[4827]: /bin/sh: 1: ps: not found
```

Reply:
Since @mloviska stated that it's not necessary anymore, I'm OK with removing it. Thanks for the clarification! :+1: